src/components/layout/nav/nav-interim.tsx

```diff
@@ -46,48 +46,69 @@ const RightSection = styled(Flex)`
+                            </LogoWrapper>
+                        </Branding>
+                    </DesktopWrapper>
 
-                <MobileWrapper>
-                    <LogoLinkMobile
-                        to={interim_type ? `/interim/${interim_type}/` : `/landing/`}
-                        aria-label="Home"
-                    >
-                        <Flex ai="center">
-                            <img src={LogoOnly} alt="deriv logo" width="115" height="19.4" />
-                            <LogoDescription ai="center">
-                                <Line />
-                                <img src={GetTrading} alt="get trading" width="120" height="17" />
-                            </LogoDescription>
-                        </Flex>
-                    </LogoLinkMobile>
-                </MobileWrapper>
-            </Flex>
+                    <MobileWrapper>
+                        <LogoLinkMobile
+                            to={interim_type ? `/interim/${interim_type}/` : `/landing/`}
+                            aria-label="Home"
+                        >
+                            <Branding>
+                                <Flex ai="center">
+                                    <img
+                                        src={LogoOnly}
+                                        alt="deriv logo"
+                                        width="115"
+                                        height="19.4"
+                                    />
+                                    <LogoDescription ai="center">
+                                        <Line />
+                                        <img
+                                            src={GetTrading}
+                                            alt="get trading"
+                                            width="120"
+                                            height="17"
+                                        />
+                                    </LogoDescription>
+                                </Flex>
+                            </Branding>
+                        </LogoLinkMobile>
+                    </MobileWrapper>
+                </Flex>
 
-            <RightSection jc="flex-end" ai="center">
-                {!landing_type && <LanguageSwitcher has_short_name />}
-                {!landing_type && (
-                    <StyledLinkButton secondary to="/">
-                        {localize('Explore Deriv.com')}
-                    </StyledLinkButton>
-                )}
-            </RightSection>
-        </Container>
-    </NavTemplate>
-)
+                <RightSection jc="flex-end" ai="center">
+                    {!landing_type && <LanguageSwitcher has_short_name />}
+                    {!landing_type && (
+                        <StyledLinkButton secondary to="/">
+                            {localize('Explore Deriv.com')}
+                        </StyledLinkButton>
+                    )}
```

Comment:
```suggestion
                    {!landing_type && (
                        <>
                            <LanguageSwitcher has_short_name />
                            <StyledLinkButton secondary to="/">
                                {localize('Explore Deriv.com')}
                            </StyledLinkButton>
                        </>
                    )}
```
This two separate conditions can be merged together

Reply:
okay